CHANGELOG.md

```diff
@@ -6,10 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## [1.2.0-rc.2] - 2024-06-14
+
+> TRUE or FALSE, it's a problem!
 
+### Important Notes
  * libcontainer/cgroups users who want to manage cgroup devices need to explicitly
    import libcontainer/cgroups/devices. (#3452, #4248)
+ * If building with Go 1.22.x, make sure to use 1.22.4 or a later version.
+   (see #4233 for more details)
+
+### Added
+
+ * CI: add actuated-arm64. (#4142)
+
+### Fixed
+
+ * Set the default value of CpuBurst to nil instead of 0. (#4210, #4211)
+ * runc exec: fix setting rlimit_nofile. (#4195, #4265, #4282, #4290)
+ * libct/cg/fs: fix setting rt_period vs rt_runtime. (#4094, #4258)
+ * script/*: fix gpg usage wrt keyboxd. (#4189)
+ * fix a debug msg for user ns in nsexec. (#4311)
+ * libct/cg: write unified resources line by line. (#4186)
+ * libct.Start: fix locking, do not allow a second container init. (#4271)
+ * Fix tests in debian testing (mount_sshfs.bats) - minimal fix. (#4245)
+ * fix checkpoint/restore tests on actuated-arm64. (#4276)
```

Comment:
We can try to start with capital letter always (except when it is runc, a path in the repo, etc.). It seems we've been doing that in the past.